src/app/app.module.ts

```diff
@@ -161,15 +163,17 @@ import { ClientEffects } from './components/shared-components/store/effects/clie
     MatSnackBarModule,
     ClipboardModule,
     StoreModule.forRoot({
+      app: loginReducer,
       token: loginReducer, 
-      clients: clientReducer
+      clients: clientReducer,
+      employeeProfileDetails : employeeProfileDetailsReducer
     }),
-    EffectsModule.forRoot([LoginEffects, ClientEffects]),
+    EffectsModule.forRoot([LoginEffects,EmployeeProfileDetailsEffects]),
```

Comment:
Fix: ClientEffects  should not be replaced by EmployeeProfileDetailsEffects instead we should add the new effects.

Reply:
done